packages/manager/apps/pci/package.json

```diff
@@ -14,7 +14,7 @@
     "@ovh-ux/manager-cloud-styles": "^0.3.0-alpha.1",
     "@ovh-ux/manager-config": "^0.2.0",
     "@ovh-ux/manager-core": "^5.3.0",
-    "@ovh-ux/manager-pci": "^0.18.0",
+    "@ovh-ux/manager-pci": "^0.19.0",
```

Comment:
:question: hmmm something went wrong?
```sh
$ npm info @ovh-ux/manager-pci | grep latest
latest: 0.18.10
```

Reply:
Yes, but in https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/package.json#L3

```json
  "name": "@ovh-ux/manager-pci",
  "version": "0.19.0",
```

Reply:
```bash
$ npm info @ovh-ux/manager-pci | grep latest
latest: 0.19.0
```